signal/src/msb_32.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "signal/src/msb.h"
+
+namespace tflite {
+namespace tflm_signal {
+
+uint32_t MostSignificantBit32(uint32_t x) {
+#if defined(__GNUC__)
```

Comment:
Can we file a bug & TODO here to see if we can allow `__builtin_clz` to be used in more cases here? It's more efficient than the other implementation, and other compilers support it as well.

Reply:
done